lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -346,12 +346,21 @@
           <h2 class="heading -emphasized -inverse"><span><?php print t('Step 4: Prove It'); ?></span></h2>
         </div>
       </div>
+
       <div class="container__row">
           <div class="container__block">
             <h3 class="inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
             <?php if (isset($reportback_copy)): ?>
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
+
+            <h3 class="inline-alt-text-color"><?php print t('Helpful Tips'); ?></h3>
+            <?php if (isset($helpful_tips)): ?>
+              <div class="with-lists">
```

Comment:
Wait.... but does that put the `ul` and the `li`s inside the `<p>...</p>`?!? Confused...